ocs_ci/ocs/node.py

```diff
@@ -2535,6 +2539,18 @@ def consumer_verification_steps_after_provider_node_replacement():
             )
             return False
 
+    log.info("Check if we can execute a ceph command from the consumer")
+    if not pod.wait_for_ceph_cmd_execute_successfully(timeout=300):
+        log.info("Try to restart the ocs-operator pod")
```

Comment:
Please mention the bug ID. This step can be removed after the bug fix.

Reply:
Yes, that will do.